### PR TITLE
[dashboard] Improve awareness for automatic workspace deletion due to garbage collection

### PR DESCRIPTION
**Pull request by ghuntley**
Progresses https://github.com/gitpod-io/gitpod/issues/2531 but the language chosen does not resolve the following aspect:

> **We want to educate users about using fresh workspaces for new tasks** but also help them understand how to use the product efficiently as well as provide useful product features that allow users to have more control over content and more.

So nit away :)

### CODE REVIEW

File: components/dashboard/src/components/workspaces.tsx [hunk 1]
```diff
@@ -273,7 +273,7 @@ class Workspaces extends React.Component<WorkspacesProps, WorkspacesState> {
                 <Grid item xs={12}>
                     <Paper className="gc-message">
                         <Typography component="p" >
-                            Unused workspaces are automatically deleted after {this.state.configuration.daysBeforeGarbageCollection} days of inactivity. <a href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection">Learn more.</a>
+                            Unused workspaces are automatically deleted after {this.state.configuration.daysBeforeGarbageCollection} days of inactivity. To prevent a workspace from being deleted, you can pin it. <a href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection">Learn more.</a>
```
Comment (gtsiolis) [inline components/dashboard/src/components/workspaces.tsx:276]: **suggestion**: This will significantly increase the sentence length beyond the common standards for readability (50-70 characters per line). What do you think of updating this with something like the following?

We can also remove trailing comma from the _Learn more_ help link as in https://github.com/gitpod-io/gitpod/pull/2536.

```suggestion
                            Unpinned workspaces that have been stopped for for than {this.state.configuration.daysBeforeGarbageCollection} days will be automatically deleted. <a href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection">Learn more</a>
```

| BEFORE | AFTER | PROPOSED |  
|-|-|-|
| alert-before | alert-after | alert-proposed |

**Review by gtsiolis — COMMENTED** [review]
Thanks for contributing @ghuntley! 🏀 

The plan is to revamp this section and make this a non-dismissable alert as described in https://github.com/gitpod-io/gitpod/issues/2531 but we can still improve the copy here before updating the layout.

Left a suggestion below, let me know what you think. 🤓

File: components/dashboard/src/components/workspaces.tsx [hunk 2]
```diff
@@ -273,7 +273,7 @@ class Workspaces extends React.Component<WorkspacesProps, WorkspacesState> {
                 <Grid item xs={12}>
                     <Paper className="gc-message">
                         <Typography component="p" >
-                            Unused workspaces are automatically deleted after {this.state.configuration.daysBeforeGarbageCollection} days of inactivity. <a href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection">Learn more.</a>
+                            Unpinned workspaces that have been stopped for for than {this.state.configuration.daysBeforeGarbageCollection} days will be automatically deleted. <a href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection">Learn more</a>
```
Comment (ghuntley) [inline components/dashboard/src/components/workspaces.tsx:276]: **nit:** for for.

PR isn't ready for merge yet.

**Comment by ghuntley** [comment]
> Left a suggestion below, let me know what you think. 🤓

TLDR - awesome suggestions. Can you please eyeball it one more time and then hit merge if 👍.

File: components/dashboard/src/components/workspaces.tsx (hunk 2 above)
Comment (gtsiolis) [reply components/dashboard/src/components/workspaces.tsx]: Thanks for catching this! 🙏

**Review by gtsiolis — APPROVED** [review]

**Comment by gtsiolis** [comment]
Let's merge this!

**Comment by gtsiolis** [comment]
Oh, just noticed the help link is not using the common pattern used elsewhere in the product (see _Learn more_). The suggestion in https://github.com/gitpod-io/gitpod/pull/3158#discussion_r577124215 was brought up as a future improvement and we need to tackle this across the product. I'll open a follow up PR.
